nullaway/src/main/java/com/uber/nullaway/Config.java

```diff
@@ -212,6 +220,15 @@ public interface Config {
 
   boolean checkContracts();
 
+  /**
+   * Checks if methods are annotated as being nullness preserving, allowing nullability to be passed
+   * through lambda parameters.
+   *
+   * @return true if methods annotated as being nullness preserving should skip some nullability
+   *     checks.
+   */
+  boolean checkNullnessPreserving();
```

Comment:
Since we're defining a new annotation for this use case, maybe we don't need a way to disable the checking?